compute/rn/release-information/release-notes-30-02-update2.adoc

```diff
@@ -0,0 +1,177 @@
+:toc: macro
+== 30.02 (Build 30.02.xxx) Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|30.02.xxx
+
+|Code name
+|Maxwell
+
+|Release date
+|June XX, 2023
+
+|Type
+|Maintenance Release
+
+|SHA-256
+|
+|===
+
+Review the https://docs.paloaltonetworks.com/prisma/prisma-cloud/30/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements] to learn about the supported operating systems, hypervisors, runtimes, tools, and orchestrators.
+
+//You can download the release image from the Palo Alto Networks Customer Support Portal, or use a program or script (such as curl, wget) to download the release image directly from our CDN: 
+//
+// LINK
+
+toc::[]
+
+//[#cve-coverage-update]
+//=== CVE Coverage Update
+
+[#new-features-agentless-security]
+=== New Features in Agentless Security
+
+//CWP-46871
+==== Encrypted volumes support in GCP with hub mode
+
+This feature adds the capability to scan encrypted volumes in GCP with agentless scanning when using hub mode.
+
+[#new-features-core]
+=== New Features in Core
+
+//CWP-48511
+==== Windows Server 2016
+
+Reinstating the support for Defenders on Windows 2016. For details on the extended support from Microsoft, see https://learn.microsoft.com/en-us/lifecycle/products/windows-server-2016
+
+//RLP-104114
+==== Added new NAT gateway IP addresses
+
+Prisma Cloud is adding new NAT IP addresses for the Compute SaaS Console Region in GCP. The egress IPs for connections from The Compute SaaS Console to the internet in us-east 1 (South Carolina) are:
+`34.139.64.150`
+`34.139.249.192`
+
+Make sure to add these IP addresses to your allow list.
+These IP addresses will be added to the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/get-started-with-prisma-cloud/enable-access-prisma-cloud-console#id7cb1c15c-a2fa-4072-b074-063158eeec08_idcb6d3cd4-d1bf-450a-b0ec-41c23a4d4280[documentation].
+
+//CWP-44813
+==== Added Support for Service Principals in Azure
+
+Added support for Azure Managed Identities to authenticate any Azure resources that support AD authentication without managing the credentials.
+
+To use this authentication method, add an Azure role with required permissions to scan the resources under *Manage > Cloud accounts*.
+
+
+[#new-features-host-security]
+=== New Features in Host Security
+
+
+//CWP-47766
+==== Containerd and CRI - ADD support for some docker compliance checks
+
+
+//CWP-34539
+==== Support custom compliance checks
```

Comment:
This feature might not make it to Maxwell Update2, waiting to hear back from Elvira to confirm!